pkg/k8shandler/forwarding.go

```diff
@@ -43,6 +43,9 @@ func (clusterRequest *ClusterLoggingRequest) generateCollectorConfig() (config s
 				// copy from defaults if output specific spec not present
 				if out.Type == logging.OutputTypeElasticsearch && out.Elasticsearch == nil {
 					out.Elasticsearch = defaultOutputSpecs.Elasticsearch
+					out.Secret = &logging.OutputSecretSpec{
+						Name: "fluentd",
```

Comment:
We MAY have a constant for the secret name too

Reply:
Fixed